### PR TITLE
update on host keys and claiming host

### PR DESCRIPTION
**Pull request by drmowinckels**

### CODE REVIEW

File: content/organization/events/online/index.en.md [hunk 1]
```diff
@@ -34,16 +34,17 @@ Feel free to book a slot for any type of R-Ladies events: talk/tutorial but also
 To let you run your meetups as independently as possible, we have set it up the
 following way:
 
-:slot_machine: Anyone entering any meeting will need a passcode. It is 
-in the description of the \#online\_meetups channel on the R-Ladies Organizer Slack (with 
-the Calendly link).
+:slot_machine: Anyone entering any meeting will need a passcode. This
```
Comment (maelle) [inline content/organization/events/online/index.en.md:37]: the passcode is not the host key? That's unclear to me from the diff.

File: content/organization/events/online/index.en.md [hunk 2]
```diff
@@ -56,14 +57,22 @@ to the calendly booking, e.g., book for 5:45pm and tell your meetup
...
+need to enter the host key which you'll get from \#online\_meetups channel on the 
+R-Ladies Organizer Slack.
 
 :wave: **Please remember to _end_ the meeting, not just _leave_ the meeting** 
 so that the next organizer can access the room for their meetup.
 
+:jigsaw: In short, you will need **two** passwords in order to start your meeting on
+the right foot.
+
+1. **Password** to get into the scheduled zoom booking (sent via e-mail and embedded in
```
Comment (maelle) [inline content/organization/events/online/index.en.md:71]: ooooh I see. I'd suggest using password in all places, not passcode.

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you!

File: content/organization/events/online/index.en.md (hunk 2 above)
Comment (drmowinckels) [reply content/organization/events/online/index.en.md]: reorganised the text to make this distinction earlier, and also always refer to it as passcode.

**Comment by drmowinckels** [comment]
@maelle I did some reorganisation of the text, let me know if you think that makes it clearer

**Review by maelle — APPROVED** [review]
:ok_hand:

**Pull request merged**